include/odp/api/spec/pool.h

```diff
@@ -518,6 +518,49 @@ unsigned int odp_pool_max_index(void);
  */
 int odp_pool_index(odp_pool_t pool);
 
+/**
+ * Pool statistics
+ */
```

Comment:
Pool capability and parameter is needed to enable/disable statistics counting.

Reply:
Fixed in v2 using by Introducin g odp_pool_stats_opt_t to get supported stats counters and configure it